src/test/kotlin/com/zepben/evolve/services/network/testdata/StupidlyLargeNetwork.kt

```diff
@@ -678,10 +679,14 @@ object StupidlyLargeNetwork {
         CustomerAgreement().apply { this.customer = customer }.addPricingStructure(pricingStructure)
 
         val location = Location()
-        val locationN1A1 = Location().apply { name = "n1"; mainAddress = StreetAddress("a1", TownDetail("", "")) }
-        val locationN1A2 = Location().apply { name = "n1"; mainAddress = StreetAddress("a2", TownDetail("", "")) }
-        val locationN2 = Location().apply { name = "n2"; mainAddress = StreetAddress("", TownDetail("", "")) }
-        val locationN3 = Location().apply { name = "n3"; mainAddress = StreetAddress("", TownDetail("", "")) }
+        val locationN1A1 = Location().apply { name = "n1"; mainAddress = StreetAddress("a1", TownDetail("", ""), "",
```

Comment:
with the need to wrap here can you please change it to wrap for the code lines rather than params
```
val locationN1A1 = Location().apply {
    name = "n1"
    mainAddress = StreetAddress("a1", TownDetail("", ""), "", StreetDetail("", "", "", "", "", "", ""))
}
```

Reply:
Saving then loading this will result in both the townDetail and streetDetail turning into nulls, which is the intended behaviour. This results in a test failure (specifically `testStupidlyLargeSchema()` in `DatabaseSqliteTest`). With the default values for `StreetAddress` mentioned above, should I replace these with something like `StreetAddress("a1")`? Or should the comparison test be modified?

Reply:
New revision uses `StreetAddress("a1")` and alike.